### DISCUSSION TITLE
WebStorm / PhpStorm can't see export

### DISCUSSION THREAD

**Discussion by iuvis** [discordjs/discord.js#8925] [Q&A]
### Which package is this bug report for?

discord.js

### Issue description

1. Import `GatewayIntentBits` or `Routes` from discord.js.

I had found that last versions of PhpStorm and WebStorm doesn't index some of discord.js exports, like these. It's not a critical bug, because code works correctly anyway, but maybe it's possible to fix it. 
I have tried to find some other peole who create bots with JetBrains products, but all work on VSCode (by the way, it indexes all exports). 

![image](https://user-images.githubusercontent.com/54040921/207333551-2ec8b515-fae1-4743-a321-56d235e6b49d.png)

### Code sample

```typescript
import {
  Client,
  GatewayIntentBits,
  Routes
} from 'discord.js';
```

### Package version

14.7.1

### Node.js version

18.12.1

### Operating system

Windows 10 

### Priority this issue should have

Low (slightly annoying)

### Which partials do you have configured?

Message

### Which gateway intents are you subscribing to?

Guilds, GuildMessages, MessageContent

**Comment by Jiralite** [comment]
I highly doubt it's a discord.js issue. It just seems to be a problem with the IDE?

**Comment by iuvis** [comment]
> I highly doubt it's a discord.js issue. It just seems to be a problem with the IDE?

I'm pretty sure it is a IDE's problem. But i hope maybe somebody had same issue, and can give me an advice how to fix it. I hope not everyone work in VSCode)))
